File: Python/tests/test_bart_withintrial_3par.py
import pytest

from hbayesdm.models import bart_withintrial_3par


def test_bart_withintrial_3par():
    _ = bart_withintrial_3par(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_withintrial_3par.R
context("Test bart_withintrial_3par")
library(hBayesDM)

test_that("Test bart_withintrial_3par", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_withintrial_3par(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
